projects/mf-dynamic-form/src/lib/utils/date-utils.ts

```diff
@@ -36,7 +36,8 @@ export class DateUtils{
     }
     private static getGapInMin(timeZone: string): number{
       const time = timeZone.split(":");
-      return (Number(time[0]) * 60)  + Number(time[1]);;
+      const negatif = Number(time[0]) < 0;
```

Comment:
Crée des variables de clarifications et renomme le commit à "Fix timezone gap when we have a négatif timezone"

```
const hoursToMinutes = Number(time[0]) * 60;
const minutes = Number(time[1]);

const negatifTimezoneGap = hoursToMinutes < 0;
return hoursToMinutes + minutes  * (negatifTimezoneGap  ? -1 : 1);

```